src/FilterRow.tsx

```diff
@@ -27,7 +29,11 @@ function FilterRow<R, SR>({
   }
 
   return (
-    <div className="rdg-filter-row">
+    <div
+      role="row"
+      aria-rowindex={ariaRowIndex}
```

Comment:
Can't we only show 1 filter row?
```suggestion
      aria-rowindex={2}
```